experimental/handoff-library/csharp_dotnetcore/Microsoft.Bot.Builder.HandoffBot/AdapterWithErrorHandler.cs

```diff
@@ -0,0 +1,42 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
can this go in a samples folder?
something like...

experimental/handoff/csharp/samples

Reply:
done